packages/stellar-axelar-std/src/tests/storage.rs

```diff
@@ -7,58 +7,62 @@ use crate::contractstorage;
 #[contract]
 pub struct Contract;
 
-#[contractstorage]
-pub enum DataKey {
-    #[instance]
-    #[value(u32)]
-    Counter,
-
-    #[persistent]
-    #[value(String)]
-    Message { sender: Address },
-
-    #[temporary]
-    #[value(Address)]
-    LastCaller { timestamp: u64 },
-
-    #[persistent]
-    #[value(bool)]
-    Flag { key: String, owner: Address },
+mod storage {
+    use super::*;
+
+    #[contractstorage]
+    enum DataKey {
+        #[instance]
+        #[value(u32)]
+        Counter,
+
+        #[persistent]
+        #[value(String)]
+        Message { sender: Address },
+
+        #[temporary]
+        #[value(Address)]
+        LastCaller { timestamp: u64 },
+
+        #[persistent]
+        #[value(bool)]
+        Flag { key: String, owner: Address },
+    }
 }
 
 #[contractimpl]
 impl Contract {
     pub const fn __constructor() {}
 
     pub fn increment_counter(env: &Env) -> u32 {
-        let current_counter = DataKey::get_counter(env).unwrap_or(0);
+        let current_counter = storage::DataKey::get_counter(env).unwrap_or(0);
```

Comment:
```suggestion
        let current_counter = storage::counter(env).unwrap_or(0);
```
See my comment / recommended usage in the task again. `DataKey` shouldn't be exposed and `get` prefix isn't needed